playbooks/roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kibana_root: /opt/kibana
+kibana_url: "https://download.elasticsearch.org/kibana/kibana/kibana-3.1.0.tar.gz"
+kibana_sha256sum: "059a4b6b507b9ff771901d12035e499b0e8d1cae7d9e5284633e19da6c294e07"
+
+elasticsearch_http_port: 9200
+elasticsearch_vip: "{{ internal_lb_vip_address }}"
+
+kibana_apt_packages:
+  - apache2
+  - python-passlib
+  - openjdk-7-jre
+
+kibana_pip_packages:
+  - python-memcached
+  - pycrypto
+
+kibana_apache_modules:
+  - rewrite
+  - ssl
+  - proxy
+  - proxy_http
+
+kibana_debug: False
+kibana_verbose: True
+
+kibana_server_name: "{{ container_name }}"
```

Comment:
`container_name` doesn't seem like it would be standard - perhaps rather use something like an ansible fact?